### DISCUSSION TITLE
Data ingestion after races

### DISCUSSION THREAD

**Discussion by jsec** [jolpica/jolpica-f1#121] [Q&A] [answered]
Hello, jolpica team!

I'm working on a hobby project that has been relying on the Ergast database dumps as a data source. Now that the season is over and the Ergast APIs are reaching EOL, I'm looking into ways to update my data sources beginning next season. 

I didn't find any information on the Ergast site regarding _how_ the data was being pulled. If I may ask, what is jolpica's planned implementation? I'm struggling to find ideas for how to accomplish that. 

(For full transparency: this project is for my own edification, I have no intention of delivering it as a competing project).

**Comment by jolpica** [comment, marked as answer]
We're going to be primarily get our data from the [PDFs the FIA publish](https://www.fia.com/documents/championships/fia-formula-one-world-championship-14/season/season-2024-2043).
@harningle has made some some scripts to parse this data [here](https://github.com/harningle/fia-doc) if you are interested in finding out more.

**Reply by jsec** [reply]
Exactly what I was looking for, thank you!
